From 8423d7eadd9aea54d5511fb4e3d811ffc97c5a87 Mon Sep 17 00:00:00 2001
From: Tiago Vignatti <tiago.vignatti@intel.com>
Date: Mon, 8 Jul 2013 19:42:09 -0300
Subject: [PATCH 4/7] WIP: Fix Ozone EGLNativeDisplayType

Use a pointer for storing the display type instead an integer, which may not
fit the value of a native display number.

Signed-off-by: Tiago Vignatti <tiago.vignatti@intel.com>
---
 third_party/khronos/EGL/eglplatform.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/third_party/khronos/EGL/eglplatform.h b/third_party/khronos/EGL/eglplatform.h
index c7d3d7b..ed5aab2 100644
--- a/third_party/khronos/EGL/eglplatform.h
+++ b/third_party/khronos/EGL/eglplatform.h
@@ -96,7 +96,7 @@ typedef void*                           EGLNativeDisplayType;
 
 #elif defined(USE_OZONE)
 
-typedef int      EGLNativeDisplayType;
+typedef intptr_t EGLNativeDisplayType;
 typedef intptr_t EGLNativeWindowType;
 typedef intptr_t EGLNativePixmapType;
 
-- 
1.7.9.5

